### PR TITLE
Est script streams

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR adds EST Stream information.

Ashish - I wasn't sure how much of the info about streaming data to remove. Please read the topic carefully.

### CODE REVIEW

File: docs/photoniq/est/est-settings.md [hunk 1]
```diff
@@ -72,3 +72,17 @@ Define how EST handles cookies.
 ### Origin Settings
 
 In the **Origin** field, enter the origin that will connect to the EST service.
+
+### Stream
+
+This option is only available if streams are enabled on your account.
```
Comment (ashish-macrometa) [inline docs/photoniq/est/est-settings.md]: PhotonIQ EST customer is agnostic to GDN, hence we just need to put general idea of `debug tracking data` feature EST is offering

File: docs/photoniq/est/manage-est-streams.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+The Stream feature in Edge Side Tagging (EST) is designed to enhance your tracking and debugging capabilities for third-party tags. This guide explains how to use this feature for debugging and analytics.
+
+:::note
+Keep the Stream feature enabled for debugging purposes only when necessary to minimize performance impact.
+:::
+
+## Understanding the EST Stream Feature
+
+Turning on EST Stream feature provides access to stream data in real time and the Debug tab in the EST dashboard.
```
Comment (ashish-macrometa) [inline docs/photoniq/est/manage-est-streams.md]: EST Stream feature is not about access to stream data , instead its for debug purpose only

File: docs/photoniq/est/manage-est-streams.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Use EST Stream for:
+
+- **Debugging** - Quickly identify and resolve issues with third-party tags by monitoring the real-time data in the Debug tab.
+- **Analytics Integration** - Feed the stream data into analytics platforms to enhance your understanding of user behavior and tag performance.
```
Comment (ashish-macrometa) [inline docs/photoniq/est/manage-est-streams.md]: we need to remove this point 
- **Analytics Integration** - Feed the stream data into analytics platforms to enhance your understanding of user behavior and tag performance.

File: docs/photoniq/est/manage-est-streams.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+- **Custom Monitoring** - Set up custom alerts or dashboards based on specific events or metrics of interest from the stream data.
+
+EST Stream is built on Macrometa Global Data Network (GDN) Streams. For more information about streams in the GDN, refer to [Macrometa Streams](../../streams/) documentation.
```
Comment (ashish-macrometa) [inline docs/photoniq/est/manage-est-streams.md]: we should remove this point 
EST Stream is built on Macrometa Global Data Network (GDN) Streams. For more information about streams in the GDN, refer to [Macrometa Streams](../../streams/) documentation.

File: docs/photoniq/est/est-settings.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/photoniq/est/est-settings.md]: Unclear what you want me to put here.

Comment (ashish-macrometa) [reply docs/photoniq/est/est-settings.md]: we need to remove this point:  "This option is only available if streams are enabled on your account..."  as EST customer is not GDN customer. Also EST customer wont be provided with GDN URL or tenant information.

Comment (diana-macrometa) [reply docs/photoniq/est/est-settings.md]: No, but the EST stream function needs to be enabled, right? Or did I misunderstand? I thought that an engineer had to toggle streams before the option was even visible to turn on or off.

Comment (ashish-macrometa) [reply docs/photoniq/est/est-settings.md]: Yes , EST stream function has to be enabled (using toggle), to see tracking data in debug tab.  But for this version EST is not offering EST stream subscription.

**Comment by diana-macrometa** [comment]
Updated. Anything else?

**Comment by ashish-macrometa** [comment]
@diana-macrometa  , Since use can see stream logs only on Debug tab, there is no API exposed for it, hence we need to remove this point :
Custom Monitoring - Set up custom alerts or dashboards based on specific events or metrics of interest from the stream data.

Also I did not get heading "Monitor and Debug P3 with EST Streams" as it should be "Monitor and Debug tracking data with EST Streams"

**Pull request merged**